phpcs.xml

```diff
@@ -14,8 +14,8 @@
     <rule ref="Generic.Files.LineEndings" />
     <rule ref="Generic.Files.LineLength">
         <properties>
-            <property name="lineLimit" value="125" />
-            <property name="absoluteLineLimit" value="125" />
+            <property name="lineLimit" value="119" />
+            <property name="absoluteLineLimit" value="119" />
```

Comment:
This limit feels very low for the absolute lines in a file. I would probably have gone fro 1000 or so.
Yes, any change that needs a larger file can simply bump this number. But how does the contributor know or discover this? Is it obvious in any way?